wot-wg-2023-draft.html

```diff
@@ -296,7 +296,29 @@ <h2>Scope</h2>
           <dd>Define bindings for specific protocols and payloads of interest while focusing to support ecosystems.</dd>
           <dt><strong>Interoperability Profiles</strong>
             (<a href="#profiles-deliverable">Profiles</a>):</dt>
-          <dd>Support out-of-the-box interoperabilty via a profiling mechanism.</dd>
+          <dd>
+	    Support out-of-the-box interoperabilty via a profiling mechanism.
+	  </dd>
+	  <dt>
+	    <strong>Digital Twins</strong>
+            (<a href="#profiles-deliverable">Profiles</a>, 
+	     <a href="#thing-description-deliverable">Thing Description</a>):
+	  </dt>
+          <dd>
+	    Enable using WoT for describing digital twins and structures thereof, 
+	    i.e. virtual things that are an abstraction of real world devices and services.
+	    This includes management of historical data, intermittent connectivity and twin graphs.
```

Comment:
historical data is covered by another work item but I guess it is fine to repeat. However, the other two subitems are a bit unclear to me.

Intermittent connectivity is already solved with MQTT. Do you have something specific in mind?

What do you mean by twin graphs?